controllers/reviewController.js

```diff
@@ -0,0 +1,13 @@
+const {
+    fetchReview
+} = require('../models/reviewModel')
+
+exports.getReview = (req, res, next) => {
+    const {review_id} = req.params
+
+    fetchReview(review_id)
+    .then(review => {
+        res.send(review)
```

Comment:
We need to be specifying the correct status code here. Express will send 200 by default, but we should be writing it explicitly here so that other devs know that it is sending a 200